control_hardware_cd_mdk

```diff
@@ -0,0 +1,16 @@
+Package: ni-routing-and-faulting-hardware-cd-mdk-veristand-{veristand_version}-labview-support

```

Comment:
```suggestion
Package: ni-routing-and-faulting-mdk-veristand-{veristand_version}-labview-support
```

I don't see a reason to include `hardware-cd` in the package name; do we envision a non-custom device MDK? I'm also eyeing `labview` for removal - are we envisioning a separate MDK package for non-LabVIEW files?

Reply:
I shortened this to:
ni-routing-and-faulting-mdk-veristand-{veristand_version}